src/Storages/RabbitMQ/StorageRabbitMQ.cpp

```diff
@@ -711,16 +711,15 @@ void StorageRabbitMQ::read(
     Pipes pipes;
     pipes.reserve(num_created_consumers);
 
+    uint64_t max_execution_time_ms = rabbitmq_settings->rabbitmq_flush_interval_ms.changed
```

Comment:
`max_execution_time_ms` is milliseconds, why we need to multiply `stream_flush_interval_ms` which is also has `ms ` suffix to `1000`?

Reply:
Indeed, it is wrong, I do not know why it was this way (here I just moved it from another place). Will fix.